modules/back-end/src/Infrastructure/Readiness/ReadinessExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Infrastructure.Redis;
+using Infrastructure.Kafka;
+
+namespace Infrastructure.Readiness;
+
+public static class ReadinessExtensions
+{
+    public const string ReadinessTag = "readiness";
+
+    public static IHealthChecksBuilder AddReadinessChecks(this IHealthChecksBuilder builder, IConfiguration configuration)
+    {
+        var readinessTags = new string[] { ReadinessTag };
+
+        builder.AddCheck<MongoDbReadinessCheck>("Check If MongoDB Is Available", tags: readinessTags)
```

Comment:
We also need to specify a timeout for each readiness check, for example

```suggestion
        builder
              .AddCheck<MongoDbReadinessCheck>(
                  "Check If MongoDB Is Available",
                  tags: readinessTags,
                  timeout: TimeSpan.FromSeconds(5)
        )
```

Reply:
Without this, when I test MongoDB being down, the `/health/readniness`  endpoint spends **30 seconds** to return a `503` status. And if the redis being down, it spends nealy **6 seconds** to return a `503` status.

Reply:
Timeouts have been applied and set to 5 seconds 👍